### PR TITLE
Return a 400 if the resource being requested is a video

### PR DESCRIPTION
**Pull request by JakeChampion**
Currently if a large video is requested then it crashes the application -- this pr will make it instead return a 400 response 👍

### CODE REVIEW

File: test/unit/lib/middleware/process-image-request.test.js [hunk 1]
```diff
@@ -306,6 +306,36 @@ describe('lib/middleware/process-image-request', () => {
 				});
 			});
 
+			describe('when the request to the original image returns a video', () => {
+				let scope;
+				beforeEach((done)=>{
+					scope = nock('https://ft.com').persist();
+					// Smallest mp4 video file from https://github.com/mathiasbynens/small/blob/master/Mpeg4.mp4
```
Comment (leafrogers) [inline test/unit/lib/middleware/process-image-request.test.js:313]: Nice!

File: lib/middleware/process-image-request.js [hunk 2]
```diff
@@ -155,6 +156,14 @@ function processImage(config) {
 						}
 						const file = imageResponse.data;
 
+						const type = await FileType.fromBuffer(file);
+						if (type && type.mime.startsWith('video')) {
+							const error = httpError(400);
+							error.cacheMaxAge = '5m';
+							error.skipSentry = true;
```
Comment (leafrogers) [inline lib/middleware/process-image-request.js:163]: is `skipSentry` to avoid sentry rate limiting?

Comment (JakeChampion) [reply lib/middleware/process-image-request.js:163]: Sort of yeah, by default errors are reported to sentry, but for this service if the error is due to a user error that we have handled then we don't want sentry to report it to us at all

File: test/unit/lib/middleware/process-image-request.test.js (hunk 1 above)
Comment (JakeChampion) [reply test/unit/lib/middleware/process-image-request.test.js:313]: Such a good repo :D

File: test/unit/lib/middleware/process-image-request.test.js [hunk 3]
```diff
@@ -277,7 +277,7 @@ describe('lib/middleware/process-image-request', () => {
 
 			});
 
-			describe('when the request to the original image returns hmtl', () => {
+			describe('when the request to the original image returns html', () => {
```
Comment (chee) [inline test/unit/lib/middleware/process-image-request.test.js:280]: can we add another test for when it returns hmtl?

**Review by chee — APPROVED** [review]

File: Makefile [hunk 4]
```diff
@@ -27,7 +27,7 @@ test-unit-coverage:
 
 # Run the integration tests using mocha
 test-integration:
-	mocha "test/integration/**/*.test.js" --recursive --reporter mocha-github-actions-reporter ${CI:+--forbid-only} --timeout $(INTEGRATION_TIMEOUT) --slow $(INTEGRATION_SLOW) $(INTEGRATION_FLAGS)
+	mocha "test/integration/**/*.test.js" --recursive --reporter mocha-github-actions-reporter ${CI:+--forbid-only} --bail --retries 5 --timeout $(INTEGRATION_TIMEOUT) --slow $(INTEGRATION_SLOW) $(INTEGRATION_FLAGS)
```
Comment (leafrogers) [inline Makefile:30]: TIL `--bail`

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v449.0.1 :tada:

**Pull request merged**